reledmac.dtx

```diff
@@ -13764,8 +13764,8 @@
 %    \end{macrocode}
 % \end{macro}
 % \begin{macro}{\do@Xfeet}
-% \protect\cs{do@Xfeet} is the code extending \protect\cs{@makecol} to cater for
-% the extra critical feet. 
+% \protect\cs{do@Xfeet} is the code extending \protect\cs{@makecol} to cater to
+% the extra critical feet % (do you mean "footers" or "footnotes"?). 
```

Comment:
Then it should be "footnotes". But if it's from the original author, you may wish to keep it like that for nostalgic reasons.